pages/index.vue

```diff
@@ -5,14 +5,24 @@
         <article-card-list class="article-card-list--big-main-post"/>
       </main>
       <aside class="aside-down">
-        <job-offers-widget/>
-        <action-links-widget/>
+        <show-component-wrapper name="job-offers-widget">
+          <job-offers-widget/>
+        </show-component-wrapper>
+        <show-component-wrapper name="action-links-widget">
+          <action-links-widget/>
+        </show-component-wrapper>
       </aside>
     </div>
     <aside class="aside-right">
-      <alert-widget class="aside-right__widget"/>
-      <meetup-calendar-widget class="aside-right__widget"/>
-      <projects-widget class="aside-right__widget"/>
+      <show-component-wrapper name="alert-widget">
```

Comment:
After the consideration i think that it would much more readable if we just simply use this 

https://codingexplained.com/coding/front-end/vue-js/showing-hiding-elements